### PR TITLE
Update Sectioning elements

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - Web/HTML/sectioning-elements/article.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/sectioning-elements/article.md

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/sectioning-elements/article.md?ref=7b74ea8ffb307975c46d394d82231617596f5796

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/sectioning-elements/footer.md

File: Web/HTML/sectioning-elements/footer.md [hunk 1]
```diff
@@ -12,20 +12,48 @@ category: must-know
...
+      janedoe@example.com
+    </a>.
+  </p>
+</footer>
+```
 
 ---
 ## Practice
 
 The `<footer>` element is encourage to contain which of the following?
```
Comment (lizTheDeveloper) [inline Web/HTML/sectioning-elements/footer.md]: Won't work, ordering

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/sectioning-elements/header.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/sectioning-elements/header.md

**Comment by enki-hq** [comment]
Failed parsing files: 
  - Web/HTML/sectioning-elements/aside.md: 
 > bad indentation of a mapping entry at line 16, column 3:
      -'[W3S docs for aside](https://w ... 
      ^

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/sectioning-elements/header.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/sectioning-elements/header.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/div.md
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/sectioning-elements/div.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/div.md
  - Web/HTML/sectioning-elements/main.md
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - Web/HTML/sectioning-elements/main.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/sectioning-elements/div.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/div.md
  - Web/HTML/sectioning-elements/main.md
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/sectioning-elements/div.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/div.md
  - Web/HTML/sectioning-elements/main.md
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/div.md
  - Web/HTML/sectioning-elements/main.md
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/div.md
  - Web/HTML/sectioning-elements/main.md
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/div.md
  - Web/HTML/sectioning-elements/main.md
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/div.md
  - Web/HTML/sectioning-elements/main.md
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/main.md
  - Web/HTML/sectioning-elements/nav.md

**Comment by enki-hq** [comment]
Rule insight:headline failed for files: 
  - Web/HTML/sectioning-elements/nav.md

**Comment by lizTheDeveloper** [comment]
This has been migrated to #766

**Pull request closed without merging**